### ISSUE TITLE
Does it have a easy way to add new annotation associate with old one except for implement annotationfinder?

### ISSUE THREAD

**Issue by wangyanstorm** [junit-team/junit#1246]
Does it have an easy way to add new annotation except for implement annotationfinder?

**Comment by kcooney** [comment]
Are you asking if JUnit will allow you to define your own annotations to replace the built-in ones? If so, the answer is it would require a custom runner.

If that wasn't what you were asking, could you rephrase your question?

**Comment by jlink** [comment]
BTW, in JUnit5 this can be easily done through meta annotations.
